docs/src/cli.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: cli
-title: "Command Line Interface"
```

Comment:
can you search over the project and replace it in bulk? We have `Command Line Interface` a bunch of times.